Common/TableProducer/qVectorsTable.cxx

```diff
@@ -185,10 +185,10 @@ struct qVectorsTable {
         if (cfgFT0CCorr->size() < 48) {
           LOGF(fatal, "No proper correction factor assigned for FT0C");
         } else {
-          cfgCorr.push_back(cfgFT0CCorr);
+          cfgCorr.push_back(std::make_shared<std::vector<float>>(cfgFT0CCorr));
         }
       } else {
-        cfgCorr.push_back(*(objft0c));
+        cfgCorr.push_back(std::make_shared<std::vector<float>>(*(objft0c)));
```

Comment:
I think there is still a leak here because the object returned by getForTimeStamp is not deleted...